docs/src/dev-documentation.md

```diff
@@ -3,26 +3,27 @@
 ## Build command cheat sheet
 
 ```bash
-# Normal build
-cd src/kani-compiler
+# Build all packages in the repository
 cargo build
 ```
+
 ```bash
-# Full regression suite
+# Full regression suite (does not run bookrunner)
 ./scripts/kani-regression.sh
 ```
 ```bash
 # Delete regression test caches
-rm -r build/x86_64-unknown-linux-gnu/tests/
+rm -r build
```

Comment:
Why? This would delete not only the test caches, but also previous `bookrunner` reports (and maybe more).